tools/ci_import

```diff
@@ -0,0 +1,122 @@
+#!/bin/env python
+
+import argparse
+import json
+import os
+import sys
+import zipfile
+import tempfile
+from os import path
+
+import gitlab
+
+IMI_DIRECTORY = "generated-image-infos"
+MANIFEST_DB_PROJECT_ID = 36844106
+
+
+def list_jobs(pipeline, token):
+    """
+    Connects to the gitlab API and retrives the list of the jobs contained in a
+    given pipeline.
+    """
+    gl = gitlab.Gitlab("https://gitlab.com", oauth_token=token)
+    project = gl.projects.get(MANIFEST_DB_PROJECT_ID)
+    pipeline = project.pipelines.get(pipeline)
+    return pipeline.jobs.list(all=True), project
+
+
+def get_job_dir(tempdir, job):
+    """
+    Returns a directory where to store a job's artifacts
+    """
+    directory = path.join(tempdir, str(job.get_id()))
+    os.makedirs(directory, exist_ok=True)
+    return directory
+
+
+def download_artifacts(directory, project, job):
+    """
+    Download an artificact.zip if available for a given job and extract it.
+    Can raise an exception if no artifacts are available.
+    """
+    filename = os.path.join(directory, "artifact.zip")
+    with open(filename, "wb") as f:
+        project.jobs.get(job.get_id()).artifacts(streamed=True, action=f.write)
+    z = zipfile.ZipFile(filename)
+    z.extractall(directory)
+    print(f"    downloading artifact.zip to {filename} for job {job.get_id()}")
+
+
+def download_image_infos(job_dir, project, job):
+    """
+    Downloads the image info and return their path as a list
+    """
+    try:
+        download_artifacts(job_dir, project, job)
+    except gitlab.exceptions.GitlabGetError as e:
+        print(f"    {e} ⚠️")
+        return []
+    image_info_directory = os.path.join(job_dir, IMI_DIRECTORY)
+    try:
+        image_infos = os.listdir(image_info_directory)
+        print(f"    image information imported for {job.get_id()}")
+        return [os.path.join(image_info_directory, image_info) for image_info in image_infos]
+    except FileNotFoundError:
+        print(f"    no image information for {job.get_id()}")
+        return []
+
+
+def update_db(image_info_path):
+    """
+    Take a downloaded image info and write its content into the corresponding
+    test case in the database.
+    """
+    image_info = image_info_path.split("/")[-1]
```

Comment:
https://docs.python.org/3/library/os.path.html#os.path.basename would be slightly better.